src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  background-color: #fff;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 60px;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    padding-top: 10px;
+    padding-left: 50px;
+  }
+
+  &__nav {
+    display: flex;
+    flex-direction: row;
+    padding-right: 50px;
+    margin-left: 243px;
```

Comment:
Using a fixed margin-left value (`243px`) might affect the responsiveness of the layout. Consider using relative units or media queries to adjust this value for different screen sizes.